data/fonts/fab/fab.yml

```diff
@@ -1,12 +1,11 @@
 ---
 fab:
-  __font_version__: 5.11.2
+  __font_version__: 5.15.0
   500px: ""
   accessible-icon: ""
   accusoft: ""
   acquisitions-incorporated: ""
   adn: ""
-  adobe: ""
```

Comment:
Please note that the `adobe` icon was removed upstream for legal reasons: https://github.com/FortAwesome/Font-Awesome/blob/f0b80521fddaa2a2e40b00576a5b64dfb8fe7301/UPGRADING.md#5140-to-5150